specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json

```diff
@@ -0,0 +1,270 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "version": "2021-03-07"
+  },
+  "paths": {
+    "/identities": {
+      "post": {
+        "tags": [
+          "Identity"
+        ],
+        "summary": "Create a new identity.",
+        "operationId": "CommunicationIdentity_CreateIdentity",
```

Comment:
The "Identity" suffix in "CreateIdentity" is redundant, should be `CommunicationIdentity_Create`.

Reply:
![image](https://user-images.githubusercontent.com/17181592/103795222-01057600-5046-11eb-99a9-d56231730bf2.png)